.github/workflows/environments-tests.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Tests - environments
+on:
+  pull_request:
+    types: ["opened", "synchronize"]
+    paths:
+      - "environments/**.go"
+      - ".github/workflows/environments-tests.yml"
+
+jobs:
+  test-environments:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
```

Comment:
You can remove this, it only affects jobs with a `matrix` configuration.

Reply:
Ah right, thanks! Will fix these in one go 👍